pkg/cluster/template/install/local_install.sh.go

```diff
@@ -68,6 +68,8 @@ fi
 
 chmod 755 "$bin_dir/tiup"
 
+exec "$bin_dir/tiup" mirror set ${script_dir} --force
```

Comment:
Please delete the https://github.com/pingcap/tiup/pull/503/files#diff-ac4e113128741529401e4349350cd6fdR96